### DISCUSSION TITLE
Typical censoring time?

### DISCUSSION THREAD

**Discussion by apocolypse323** [silveredgold/beta-protection#48] [Q&A] [answered]
Is there a standard length of time that it takes for an image to be censored? I know it's not meant to be instant, but I am finding that I can wait for several minutes, and nothing seems to be happening. I've even left an image in the background to be censored, and after 15-20 minutes, it's still loading. I have a newer gaming laptop, so I can't imagine that it's a hardware issue.

**Comment by silveredgold** [comment, marked as answer]
Generally speaking on vaguely modern hardware, it's usually a few seconds per image. On my normal PC (a Ryzen 5000 series), the average time is around 1.5s per image.

**Reply by apocolypse323** [reply]
Sorry about that, turns out I'd installed it wrong. It's all loading fine now :)

**Comment by SpiritRetro** [comment]
what was the solution to this? as i think i may be havign the same issue
